### PR TITLE
Newint - support for moment.js

### PR DESCRIPTION
**Pull request by iambibhas**
Added support as mentioned in #1554.

### CODE REVIEW

File: share/spice/newint/newint.js [hunk 1]
```diff
@@ -6,41 +6,40 @@
...
+                id: "newint",
+                name: "Magazines",
+                data: api_result,
+                meta: {
+                    itemType: "New Internationalist magazines",
+                    sourceName: "New Internationalist",
+                    sourceUrl: 'https://digital.newint.com.au/issues'
+                },
+                normalize: function(item){
+                    var editors_letter_strip_tags = item.editors_letter_html.replace(/<h[1-6]>(.*?)<\/h[1-6]>/g, "<p>$1. </p>");
```
Comment (moollaza) [inline share/spice/newint/newint.js]: Looks like we can use `DDG.strip_html` and `DDG.strip_href` here?

File: share/spice/newint/newint.js [hunk 2]
```diff
@@ -6,41 +6,40 @@
...
+                    editors_letter_strip_tags = editors_letter_strip_tags.replace(/<a(.*?)>(.*?)<\/a>/g, "$2");
+                    return {
+                        release_date: moment(item.release).format('MMMM YYYY'),
```
Comment (moollaza) [inline share/spice/newint/newint.js]: :+1:

File: share/spice/newint/newint.js (hunk 1 above)
Comment (iambibhas) [inline share/spice/newint/newint.js]: It seems to replace just the header tags and anchors, and leaves everything else. So maybe `strip_href`, not `strip_html`, that is if we want to keep the formatting as it is. Or we could just pass it through `strip_html` and show the text only.

Comment (moollaza) [inline share/spice/newint/newint.js]: @iambibhas wow I just noticed a few lines down we're running it through `DDG.strip_html` so I think these two lines are actually redundant. We should be able to simply remove them :)

Comment (iambibhas) [inline share/spice/newint/newint.js]: Ugh, I also totally missed it. :| Yeah, we can just remove them.

**Pull request merged**